cmd/client-fs.go

```diff
@@ -987,10 +987,11 @@ func (f *fsClient) listRecursiveInRoutine(contentCh chan *clientContent) {
 }
 
 // MakeBucket - create a new bucket.
-func (f *fsClient) MakeBucket(region string, ignoreExisting bool) *probe.Error {
+func (f *fsClient) MakeBucket(region string, ignoreExisting, withLock bool) *probe.Error {
 	// TODO: ignoreExisting has no effect currently. In the future, we want
 	// to call os.Mkdir() when ignoredExisting is disabled and os.MkdirAll()
 	// otherwise.
+	// NOTE: withLock=true has no meaning here.
```

Comment:
```suggestion
	// withLock is currently ignored. Send false.
```

Add this to the function description instead of here.

Reply:
This is not correct because top level `mainMakeBucket()` unaware what client it created. Hence it passes `withLock` flag to underneath `Client.MakeBucket()` irrespective of the value.